docs/data/data-grid/row-dimensions/row-dimensions.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Data Grid - Row dimensions
+---
+
+# Data grid - Row dimensions
```

Comment:
What about renaming to "Row height"? The row has only one dimension: height.

Reply:
Would you rename the "Column dimensions" page "Columns width" ?

Reply:
I think "Row height" makes more sense here. We can discuss renaming "Column dimensions" later.